src/pudl/transform/eia861.py

```diff
@@ -456,18 +456,26 @@
     "transportation"
 ]
 
+FUEL_CLASSES = [
```

Comment:
I'll echo my comment from constants here that it would probably be good to have these classes be a part of the same fuel type codes we use in EIA and FERC. We don't have common list of all fuel type codes rn, but we typically use the `fuel_type_code_pudl` column as the column that can help the data sets talk to each other.  

These are the three dictionaries in constants that currently map the various fuel codes/strings to more universal codes:
- `fuel_type_eia860_simple_map`
- `fuel_type_eia923_gen_fuel_simple_map`
- `ferc1_fuel_strings`
It makes sense that there are new codes here bc we basically have no renewable data in the other data sets.

Reply:
gocha - should I make a `fuel_type_eia861_map` ?

Reply:
These maps are mostly used to map the various weird strings to our standardized codes. I don't think you need that because the fuel type here is first assigned in our column names, which you have standardized. So I don't think a map is required! Mostly I'm hoping we ensure that the names in 861 are the same, which I think they are so far. And perhaps we should change the name to the column so it is easy to see that they can/should be compared.